packages/fxa-content-server/server/lib/routes/post-csp.js

```diff
@@ -22,21 +22,14 @@ const BODY_SCHEMA = {
     .object()
     .keys({
       // CSP 2, 3 required
-      // `eval` and `inline` are specified in CSP 3 and sent by Chrome
-      'blocked-uri': URL_TYPE.allow('')
-        .allow('asset')
-        .allow('blob')
-        .allow('data')
-        .allow('eval')
-        .allow('inline')
-        .allow('self')
+      'blocked-uri': STRING_TYPE.allow('')
```

Comment:
Let's let more things go through. It just hits a log table anyways. I'd rather see the csp violation errors people are having than be accidentally exclude something and not know about it.